### ISSUE TITLE
Bug: Select All Then Deselect Row Does Not Deselect Header Checkbox

### ISSUE THREAD

**Issue by jdcalkins** [jdcalkins/ng2-data-table#12]
The select/deselect all checkbox in the header does not uncheck when a checked row is unchecked.

**Comment by jdcalkins** [comment]
Solution is to set a template variable on mfRowSelectorHead

``` typescript
<mfRowSelectorHead #mfRowSelectorHead ></mfRowSelectorHead>
```

Then in the mfRowSelector.selectEntity emitter, add a function call to the template variable with isChecked set to false.

``` typescript
<mfRowSelector (selectEntity)="mf.addRemoveSelectedEntity($event); mfRowSelectorHead.isChecked = false;"></mfRowSelector>

```

**Comment by jdcalkins** [comment]
Bug resolved with #13
